servers/rendering/rasterizer_rd/rasterizer_canvas_rd.cpp

```diff
@@ -1671,12 +1721,22 @@ void RasterizerCanvasRD::ShaderData::set_code(const String &p_code) {
 
 		} break;
 		case BLEND_MODE_MIX: {
+			/*
```

Comment:
Is this commented code worth keeping? Old code will still be in the git history.